frontend/src/types.ts

```diff
@@ -294,13 +294,22 @@ export interface PersonType {
 }
 
 export interface CohortGroupType {
+    id: string
     days?: string
     action_id?: number
+    event_id?: string
+    label?: string
+    count?: number
+    count_operator?: string
     properties?: Record<string, any>
+    matchType: MatchType
 }
 
+export type MatchType = 'entities' | 'properties'
```

Comment:
We also use these as constants, should we just declare them in a single place to simplify?